darts/models/forecasting/block_rnn_model.py

```diff
@@ -311,6 +311,30 @@ def encode_year(idx):
         ----------
         .. [1] T. Kim et al. "Reversible Instance Normalization for Accurate Time-Series Forecasting against
                 Distribution Shift", https://openreview.net/forum?id=cGDAkQo1C0p
+
+        Examples
+        --------
+        >>> from darts.datasets import AirPassengersDataset
+        >>> from darts.models import BlockRNNModel
+        >>> from darts.utils.timeseries_generation import datetime_attribute_timeseries as dt_attr
+        >>> series = AirPassengersDataset().load()
+        >>> # optionnaly, one-hot encode the months as a past covariates
```

Comment:
we can use the same as my example for future covariates with `add_length=0`
also, no need to abbreviate `datetime_attribute_timeseries`